styles/main.css

```diff
@@ -205,10 +184,70 @@ ul {
   position: absolute;
   top: 0;
   bottom: 0;
-  height: 30px;
+  left: 0.5rem;
```

Comment:
?

Reply:
They are positioning values (left, right, top, bottom). When an element is positioned absolutely, any positioning value is going to move the element that many units away from its parent's left, right, top, or bottom side.
https://developer.mozilla.org/en-US/docs/Web/CSS/position

Reply:
Got it. Sounds good. Thank you !